script/server

```diff
@@ -31,7 +31,7 @@ fi
 echo "==> Starting the server..."
 
 if [ "$MOCK_API" = true ]; then
-  ACCREDITED_PROGRAMMES_API_URL=http://localhost:9099 npm run start:dev
+  ACCREDITED_PROGRAMMES_API_URL=http://localhost:9099 PRISONER_OFFENDER_SEARCH_URL=http://localhost:9099 npm run start:dev
 else
-  npm run start:dev
+  PRISONER_OFFENDER_SEARCH_URL=http://localhost:9099 npm run start:dev
```

Comment:
If this is the same across mocked and unmocked API states, shouldn't it just be in `.env` and `.env.example`?

Reply:
If we're running against dev (with `npm run start:dev`) there'll be a clash with our local `.env` file which points at the dev environment. We'll probably want to add some new flag functionality here to specify which environment we're running. I'll add a card to do that in future.